lib/ml/regression.hpp

```diff
@@ -64,6 +64,9 @@ class Regression {
 
     // query parameters
     int get_num_param() { return param_list_.get_num_param(); }  // get parameter size
+
+    // TODO(GUOYuyong): add get_param function to return vector for bindings
+    ParamT get_param() { return param_list_; }
```

Comment:
Does this need to be modify by the caller? If not, `inline const ParamT& get_param()` is better.